harness/determined/common/experimental/experiment.py

```diff
@@ -51,6 +51,10 @@ def __init__(self, raw: Any):
 
 class ExperimentReference:
     """
+    An ExperimentReference object is usually used in conjunction with
```

Comment:
"used in conjunction with" doesn't make much sense, I think you should say 'is usually obtained from' or 'is usually returned by' or something.